src/test/java/core/basesyntax/FruitShopTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.RecordDao;
+import core.basesyntax.dao.RecordDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.services.ReportService;
+import core.basesyntax.services.ReportServiceImpl;
+import core.basesyntax.services.actions.ActionHandler;
+import core.basesyntax.services.actions.BalanceHandler;
+import core.basesyntax.services.actions.IncreaseHandler;
+import core.basesyntax.services.actions.PurchaseHandler;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitShopTest {
```

Comment:
have you read the checklist carefully? there is an instruction here about how we can exclude `Main` class (or any other class) from test coverage check